kolibri/deployment/default/settings/base.py

```diff
@@ -188,8 +188,14 @@
     "la": {
         "bidi": False,
         "code": "la",
-        "name": "Spanish (Latin America)",
-        "name_local": "Español América Latina",
+        "name": "Spanish",
```

Comment:
for the English variation of the name, I think it's actually helpful to leave as "Spanish (Latin America)". This gets used as a tooltip. I'll push an update

Reply:
Sounds good, I was trying to be consistent, but I agree that for the tooltip is OK to have it in full.